test_framework/scripts/longhorn-setup.sh

```diff
@@ -54,6 +54,48 @@ install_csi_snapshotter_crds(){
 }
 
 
+install_rancher() {
+
+  RANCHER_HOSTNAME=`cat "${TF_VAR_tf_workspace}/load_balancer_url"`
+  RANCHER_BOOTSTRAP_PASSWORD='p@ssw0rd'
+
+  kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.crds.yaml
+  kubectl create namespace cert-manager
+  helm repo add jetstack https://charts.jetstack.io
+  helm repo update
+  helm install cert-manager jetstack/cert-manager --namespace cert-manager --version v1.4.0
+  kubectl get pods --namespace cert-manager
+
+  helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+  kubectl create namespace cattle-system
+  helm install rancher rancher-latest/rancher --namespace cattle-system --set bootstrapPassword="${RANCHER_BOOTSTRAP_PASSWORD}" --set hostname="${RANCHER_HOSTNAME}" --set replicas=3 --set ingress.tls.source=letsEncrypt --set letsEncrypt.email=yang.chiu@suse.com
+  kubectl -n cattle-system rollout status deploy/rancher
+}
+
+
+get_rancher_api_key() {
+  python3 "${TF_VAR_tf_workspace}/scripts/rancher/webdriver/main.py" "${RANCHER_HOSTNAME}" "${RANCHER_BOOTSTRAP_PASSWORD}"
+  RANCHER_ACCESS_KEY=`cat "${PWD}/access_key"`
+  RANCHER_SECRET_KEY=`cat "${PWD}/secret_key"`
+}
+
+
+install_longhorn_by_rancher() {
+  terraform -chdir="${TF_VAR_tf_workspace}/scripts/rancher/terraform" init
+  terraform -chdir="${TF_VAR_tf_workspace}/scripts/rancher/terraform" apply \
+            -var="api_url=https://${RANCHER_HOSTNAME}" \
+            -var="access_key=${RANCHER_ACCESS_KEY}" \
+            -var="secret_key=${RANCHER_SECRET_KEY}" \
+            -var="longhorn_git_repo=${LONGHORN_RANCHER_CHART_REPO}" \
```

Comment:
Suggest renaming to `rancher_chart_git_repo` and the other is `rancher_chart_git_branch`

Reply:
Modified.